app/models/event.rb

```diff
@@ -5,4 +5,5 @@ class Event < ActiveRecord::Base
   validates_presence_of :started_at
 
   belongs_to :owner, foreign_key: 'user_id', class_name: 'User'
```

Comment:
Sorry I didn't catch this before, but we do associations before validations in our model. A nemonic to remember this could be that the word `associations` comes before `validations` in the dictionary